chpl/chpl-service/src/main/java/gov/healthit/chpl/realworldtesting/manager/RealWorldTestingReportService.java

```diff
@@ -173,38 +173,44 @@ private String formatUserData(UserDTO user) {
     }
 
     private RealWorldTestingReport addMessages(RealWorldTestingReport report) {
+        String plansMessage = "";
         if (isWithdrawn(report.getCurrentStatus())) {
-            report.setRwtPlansMessage(errorMsg.getMessage("realWorldTesting.report.listingWithdrawnMessage"));
+            plansMessage = errorMsg.getMessage("realWorldTesting.report.listingWithdrawnMessage");
         } else if (isRwtPlansEmpty(report)) {
             if (BooleanUtils.isTrue(report.getIcs())
                     && (arePlansLateWarning(report.getRwtEligibilityYear()) || arePlansLateError(report.getRwtEligibilityYear()))) {
-                report.setRwtPlansMessage(errorMsg.getMessage("realWorldTesting.report.eligibleByIcs.missingPlansError",
-                        report.getRwtEligibilityYear().toString()));
+                plansMessage =errorMsg.getMessage("realWorldTesting.report.eligibleByIcs.missingPlansError",
```

Comment:
Checkstyle called this out, and I didn't want to approve it without doubling down on this. I'll approve the PR now, but want this changed before it gets merged
```suggestion
                plansMessage = errorMsg.getMessage("realWorldTesting.report.eligibleByIcs.missingPlansError",
```